unit-testing/src/mock-function/data/model-schema.ts

```diff
@@ -0,0 +1,43 @@
+// input-verified.ts
+import { z } from 'zod';
```

Comment:
I ❤️ zod, but wonder if this helps the example or is yet another concept someone new has to learn? Wdyt?

Reply:
Ok, I'll simplify validation.